var/spack/repos/builtin/packages/libyogrt/package.py

```diff
@@ -35,9 +36,13 @@ class Libyogrt(AutotoolsPackage):
 
     variant('scheduler', default='system',
             description="Select scheduler integration",
-            values=['system', 'slurm', 'lsf'], multi=False)
-    depends_on('slurm', when='scheduler=slurm')
+            values=['system', 'flux', 'lsf', 'slurm'], multi=False)
+    depends_on('flux-core@0.21.0:', when='scheduler=flux')
     depends_on('lsf', when='scheduler=lsf')
+    depends_on('slurm', when='scheduler=slurm')
+
+    # support for flux added in libyogrt 1.27
+    conflicts('scheduler=flux', when='@:1.24')
```

Comment:
Should `scheduler=flux` conflict with libyogurt <= v1.26, not 1.24?
```suggestion
    conflicts('scheduler=flux', when='@:1.26')
```

Reply:
Yeah, good point.  I went with 1.24, because there is no 1.25 posted and it looks like libyogrt-1.26 might also have flux support though I haven't tested it.  In that case, 1.24 is the last version that does not have flux support.

I think it's best to change this to 1.26 as you suggest.

Reply:
Fixed to conflict with versions <= 1.26.

Reply:
Ah, ok. I was going completely off the comment above the `conflicts` line